### PR TITLE
Revise ZA relevant language entries

### PR DESCRIPTION
**Pull request by lcjohnso**
Staging branch URL: https://pr-6732.pfe-preview.zooniverse.org

Matching https://github.com/zooniverse/pandora/pull/661, update language entries relevant to South Africa, taking the following actions
- Remove duplicate entries.
- Make all `-za` entries general by removing country specification (non-specific versions were missing before).
- Edit labels for ZA-relevant entries to include standard/reference language name (matching ISO language name) and local names separated by "/" character.  e.g., for `xh`, "Xhosa / isiXhosa"

Surpasses and replaces https://github.com/zooniverse/Panoptes-Front-End/pull/6731.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `npm ci` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on GitHub Actions?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/constants/languages.js [hunk 1]
```diff
@@ -77,6 +75,7 @@ const languages = [
   { value: 'moh-ca', label: 'Mohawk (Mohawk)' },
   { value: 'mn', label: 'Mongolian' },
   { value: 'ne-np', label: 'Nepali (Nepal)' },
+  { value: 'nso', label: 'Northern Sotho / Sesotho sa Leboa' },
```
Comment (eatyourgreens) [inline app/constants/languages.js:78]: Can we please not use colonial names?

**Review by eatyourgreens — CHANGES_REQUESTED** [review]
Can we use the language names that we’ve been given by our colleagues in South Africa?

File: app/constants/languages.js (hunk 1 above)
Comment (lcjohnso) [reply app/constants/languages.js:78]: As I wrote in the PR text, I'm opting to include _both_ the official ISO Language Name and the local/native name. Including both names is useful -- it resolves possible confusion in the case of "Sesotho" -- Sesotho = Southern Sotho = `st` vs. Sesotho sa Leboa = Northern Sotho = `nso`.

Comment (eatyourgreens) [reply app/constants/languages.js:78]: The ISO name is not official. Can we use the names that we’ve been asked to use by people who speak these languages.

Comment (eatyourgreens) [reply app/constants/languages.js:78]: Also, ‘native name’????

**Comment by eatyourgreens** [comment]
Here's the list of languages that we've been given:
- isiXhosa
- Sesotho
-Tshivenḓa
- Xitsonga
- isiZulu

If we could stick with those names, that would be great. If we must add the European names too, then we could maybe do something like 'Xitsonga / Tsonga' but I'm not sure what value the European names add to the project builder.

**Pull request closed without merging**